### DISCUSSION TITLE
Cant start stream from outside of LAN.  Worked before changing my router.

### DISCUSSION THREAD

**Discussion by Reyemrk** [GerryDazoo/Slinger#549] [General]
Cant start stream from outside of LAN.  It worked before November, before changing my router to one that has settings to block the slingbox from accessing the internet.  I have checked if my port is open (:8080) on my server machine.  I checked my public ip with whatsmyip.com.  And I can stream to VLC when I am on my local network.  When I connect to stream from outside of LAN on VLC or MX the stream closes immediately, and states, " Playback failed".  I have turned off my firewall on my server machine and on the router.  Ideas of what is going on here?

Thank you for any help.

Version : 4.0 Running on Windows-10-10.0.22621-SP0 pid= 12496 C:\slinger\Slinger-main\slingbox_server.exe
Using config file config.ini
Connection Manager Running on port 8080 with 10 max streams using URL slingbox.
Building page for Slingbox
Using built in default remote page definition.
Starting Streamer Thread for Slingbox
Streamer Running:  10 config.ini SLINGBOX 8080 8080 8388608
starting up on  port 8080 8388608
8080 Checking for slingbox at ('', 5201) 0
 * Serving Flask app 'slingbox_server'
 * Debug mode: off
8080 Error connecting to  ('', 5201)
Broadcast Responder Running
Server IP address is 10.0.1.11
8080 No valid slingbox ip info found in config.ini
8080 Finding Slingboxes on local network. My IP Info =  10.0.1.11
8080 Found at 10.0.1.254 5001 " u's Slingbox " FinderID 293412562C67Axx8B44480EFxxB1dd6E9 ProductID PRO
8080 Using slingbox at  ('10.0.1.254', 5001)
Streamer:  8080 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
02/28/2023, 18:46:52.491   Streaming request from ('10.0.1.14', 53509)  /slingbox
STREAM=10.0.1.14:53509:0

**Comment by GerryDazoo** [comment]
Then you've got something wrong with your port forwarding rule  TCP 8080->10.0.1.11:8080 on your router.
PS. You should change the [SERVER] port to something else to minimize the spurious connection requests from hackers.
